package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "yktotp",
+  "version": "0.1.0",
+  "description": "Get a YubiKey OTP into the browser",
+  "repository": "git@github.com:nilsalex/yktotp.git",
+  "author": "Nils Alex <nils@famalex.de>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf dist/",
+    "build": "webpack",
+    "lint": "eslint . --ext .ts,.tsx",
+    "prettier": "prettier --write ."
+  },
+  "devDependencies": {
+    "@types/chrome": "^0.0.183",
+    "@types/react": "^18.0.6",
+    "@types/react-dom": "^18.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.19.0",
+    "@typescript-eslint/parser": "^5.19.0",
+    "copy-webpack-plugin": "^10.2.4",
+    "eslint": "^8.13.0",
+    "eslint-config-prettier": "^8.5.0",
+    "prettier": "^2.6.2",
+    "react": "^18.0.0",
```

Comment:
i think react etc are not *dev* dependencies

Reply:
I can appreciate each of the three arguments laid out here https://jsramblings.com/do-dependencies-devdependencies-matter-when-using-webpack/